tv.plex.PlexDesktop.yml

```diff
@@ -10,16 +10,6 @@ finish-args:
   - --share=ipc
   - --share=network
   - --talk-name=org.freedesktop.ScreenSaver
-cleanup:
```

Comment:
Leave the top level cleanup for common stuff like /include, /lib/pkgconfig, /lib/cmake as is. Add only the per module ones to the individual modules.

If you add them all in per module, it makes the manifest long.

Reply:
moved all shared entries to the toplevel entry.